allocation/models.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from applications.models import ApplicationRound, ApplicationRoundBasket
+
+User = get_user_model()
+
+
+class AllocationRequest(models.Model):
+
+    start_date = models.DateTimeField(
+        verbose_name=_("Start time"), null=False, blank=True
+    )
+
+    end_date = models.DateTimeField(verbose_name=_("End time"), null=True, blank=True)
+
+    completed = models.BooleanField(verbose_name=_("Completed"), null=True, blank=True)
```

Comment:
should we add `default=False` and make this non nullable? Would make it easier to query for not completed ones.

Reply:
Yeah, that sounds better.